forge-gui/res/cardsfolder/upcoming/rulik_mons_warren_chief.txt

```diff
@@ -0,0 +1,12 @@
+Name:Rulik Mons, Warren Chief
+ManaCost:1 R G G
+Types:Legendary Creature Goblin
+PT:3/3
+K:Menace
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ DBDig | TriggerDescription$ Whenever CARDNAME Chief attacks, look at the top card of your library. If it's a land card, you may put it onto the battlefield tapped. If you didn't put a card onto the battlefield this way, create a 1/1 red Goblin creature token.
+SVar:DBDig:DB$ Dig | DigNum$ 1 | ChangeNum$ 1 | ChangeValid$ Card.Land | Optional$ True | RememberChanged$ True | DestinationZone$ Battlefield | DestinationZone2$ Library | LibraryPosition2$ 0 | SubAbility$ DBToken
+SVar:DBToken:DB$ Token | ConditionDefined$ Remembered | ConditionPresent$ Card.inZoneBattlefield | ConditionCompare$ EQ0 | TokenScript$ r_1_1_goblin | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHas:Ability$Token
+DeckHints:Type$Goblin
```

Comment:
this line isn't needed - since this card doesn't directly benefit from other goblins
probably add HasAttackEffect line though